powersimdata/scenario/execute.py

```diff
@@ -290,7 +291,7 @@ def create_folder(self):
         :raises IOError: if folder cannot be created.
         """
         print("--> Creating temporary folder on server for simulation inputs")
-        command = "mkdir %s" % self.TMP_DIR
+        command = "mkdir -p %s" % self.TMP_DIR
```

Comment:
Just curious, why do you need the `-p` option here

Reply:
Couple reasons - create missing parent directories, and makes it re-runnable. Initially I got an error during `prepare_simulation_input` due to the `tmp` folder not existing yet. So this fixes that, and has the nice side effect of not failing when called twice.

Reply:
Yep, I had to deal with errors due to the missing `tmp` folder in my ssh server build as well